src/mqt/bench/benchmarks/qiskit_application_finance/pricingput.py

```diff
@@ -1,3 +1,4 @@
+"""Pricing put benchmark definition."""
```

Comment:
There is a comment right below this top level one. Consider combining the two.

Reply:
Fixed it.